scenarios/arcaflow/cpu-hog/sub-workflow.yaml

```diff
@@ -79,11 +79,19 @@ steps:
       type: kubernetes
       connection: !expr $.steps.kubeconfig.outputs.success.connection
       pod:
+
         metadata:
           namespace: !expr $.input.namespace
           labels:
```

Comment:
does the namespace get created within the stress ng code? If you add the labels like this to the namespace at creation you wont have to add to every pod/step 
apiVersion: v1
kind: Namespace
metadata:
  labels:
    kubernetes.io/metadata.name: <name>
    pod-security.kubernetes.io/audit: privileged
    pod-security.kubernetes.io/enforce: privileged
    pod-security.kubernetes.io/enforce-version: v1.24
    pod-security.kubernetes.io/warn: privileged
    security.openshift.io/scc.podSecurityLabelSync: “false”
  name: <name>